README.md

```diff
@@ -94,12 +94,25 @@ airflow db init
 Create an SQLite database for the example to run with and run the DAG:
 ```
 sqlite3 /tmp/sqlite_default.db "VACUUM;"
-airflow dags test calculate_top_animations `date --iso-8601=seconds`
+airflow dags test calculate_popular_movies `date -Iseconds`
 ```
 
 Check the top five animations calculated by your first Astro DAG by running:
-```commandline
-sqlite3 /tmp/sqlite_default.db "select * from top_animation;" ".exit"
+```shell
+# The sqlite_default host is different for MAC vs. Linux
```

Comment:
:see_no_evil: I didn't realise that was the case

Reply:
I didn’t know too 😄 - It is because of the following line:

https://github.com/apache/airflow/blob/f06b3955b1d937138fb38021a6a373b94ae8f9e8/airflow/utils/db.py#L560

Which returns a different dir /var/random_nums  on my mac🤦‍♂️